examples/contrail-k8s-nested/bundle.yaml

```diff
@@ -25,6 +25,7 @@ services:
       nested_mode_config: |
         {
         "CONTROLLER_NODES": "10.0.12.20",
+        "CONTROL_NODES": "10.0.12.20",
```

Comment:
it  has sense only if it's not the equal to CONTROLLER_NODES.
let's revert this file